src/pages/test-result/test-result.html

```diff
@@ -1,25 +1,35 @@
 <ion-header>
-    <page-header [isRecording]="isRecording"></page-header>
+    <mes-header [title]='getTitle()'></mes-header>
 </ion-header>
 
 <ion-content padding class="mes-main-content">
     <ion-card class="mes-card">
-        <ion-card-header class="mes-main-card-header mes-breadcrumbs">
-            <span class="mes-header-md mes-breadcrumb-link" (click)="goBack()">
-                <ion-icon name="arrow-back"></ion-icon>
-                Back to DL25 /
-            </span>
-            <span padding-left class="mes-header-md">Florence Pearson</span>
-        </ion-card-header>
-        <ion-card-content>
-            <test-summary [faultSummaries]="faultSummaries" [testResult]="testResult" [summaryMetadata]="summaryMetadata"></test-summary>
-        </ion-card-content>
+        <div class="test-result-header-card">
+            <ion-card-header class="card-header">
+                Test requirements and actions
+            </ion-card-header>
+            <test-result-header [summaryMetadata]="summaryMetadata"></test-result-header>
+        </div>
     </ion-card>
-    <ion-card class="mes-bottom-card" text-center>
-        <ion-row align-items-center>
-            <ion-col>
-                <button [navPush]="getNextPage()" class="mes-button-primary">End debrief</button>
-            </ion-col>
-        </ion-row>
+    <ion-card class="mes-card">
+        <ion-card-content text-center [ngClass]="getTestOutcomeContentClass()">
+            <div id="test-outcome-label">
+                Test outcome
+            </div>
+            <div id="test-outcome">
+                {{testResult}}
+            </div>
+        </ion-card-content>
+        <div *ngIf="wasThereAnyFault()">
+            <ion-card-header class="card-header">
+                Faults recorded
+            </ion-card-header>
+            <ion-card-content>
+                <test-summary [faultSummaries]="faultSummaries" [testResult]="testResult" [summaryMetadata]="summaryMetadata"></test-summary>
+            </ion-card-content>
+        </div>
     </ion-card>
+    <div class="test-result-footer" text-center>
+        <button [navPush]="getNextPage()" class="mes-button-primary confirm-button">End debrief</button>
```

Comment:
Could we use the new button style as per the [component library](https://wiki.i-env.net/display/MES/Component+library)
I think the class name is `.mes-cta-button-primary`